src/js/game/GameController.js

```diff
@@ -715,7 +715,11 @@ class GameController {
   handleMoveOffPressurePlate(commandQueueItem, moveOffset) {
```

Comment:
Can this (and `GameController.prototype.handleMoveOnPressurePlate`) go away now?